cmd/flow-dps-server/README.md

```diff
@@ -0,0 +1,26 @@
+# Flow DPS Server
+
+## Description
+
+The Flow DPS Server runs on top of a DPS index to provide random access to the execution state through its API.
+Both the Flow DPS Indexer and the Flow DPS Live tool can create such an index.
+In the case of the indexer, the index is static and built from a previous spork's state.
+For the live tool, the index is dynamic and updated on an ongoing basis from the data sent from a Flow execution node.
+Access to the execution state is provided through a GRPC API.
```

Comment:
`flow-grpc-server`? and the client could be `flow-grpc-client` to make it clear that they're supposed to work together, and that the client isn't to be used with `flow-rosetta-server`.

Reply:
No, it's the DPS API. We use GRPC as the standard. There is only one DPS API now.